<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magic Cubiano</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <widget class="QLabel" name="chooseLabel">
       <property name="text">
        <string>Guitar mode. Choose string and fret for cube faces:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="uEdgeWidget" native="true">
     <property name="name" stdset="0">
      <string>U</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="rEdgeWidget" native="true">
     <property name="name" stdset="0">
      <string>R</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="lEdgeWidget" native="true">
     <property name="name" stdset="0">
      <string>L</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="fEdgeWidget" native="true">
     <property name="name" stdset="0">
      <string>F</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="dEgdeWidget" native="true">
     <property name="name" stdset="0">
      <string>D</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EdgeSettingsWidget" name="bEdgeWidget" native="true">
     <property name="name" stdset="0">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="0,100,0,0,0,0,0">
     <item row="1" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="durLineEdit">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="3">
      <widget class="QSlider" name="allStringsSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QLabel" name="durLabel">
       <property name="text">
        <string>Duration [mSec]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QPushButton" name="resetStringsButton">
       <property name="text">
        <string>Reset all strings</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="allStringsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="statusTextLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EdgeSettingsWidget</class>
   <extends>QWidget</extends>
   <header>EdgeSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
